apertium-kmr-eng.kmr-eng.dix

```diff
@@ -11963,6 +11965,7 @@
      <e><p><l>eşkere<s n="adj"/></l><r>obvious<s n="adj"/></r></p></e>
      <e><p><l>eşkere<s n="adj"/></l><r>clear<s n="adj"/></r></p></e>
      <e><p><l>nerm<s n="adj"/></l><r>suitable<s n="adj"/></r></p></e>
+     <e><p><l>nerm<s n="adj"/></l><r>tender<s n="adj"/></r></p></e>
```

Comment:
How do you know it means this?

Reply:
I got this from the Glosbe online dictionary; I can remove it, though, if you'd like.